cmd/oci/oci.go

```diff
@@ -60,13 +61,30 @@ var OCICmd = &cobra.Command{
 			os.Exit(1)
 		}
 		platform = p
+		nameMap := make(map[string]string)
```

Comment:
better variable name please - it should be related to the domain/action.

Reply:
how does `articfactName` sounds?

Reply:
sounds good